src/ert/config/analysis_config.py

```diff
@@ -41,16 +40,16 @@ def __init__(
         max_runtime: int = 0,
         min_realization: int = 0,
         update_log_path: Union[str, Path] = "update_log",
-        analysis_iter_config: Optional[AnalysisIterConfig] = None,
         analysis_set_var: Optional[List[Tuple[str, str, str]]] = None,
+        num_iterations: int = 1,
     ) -> None:
         self._max_runtime = max_runtime
         self.minimum_required_realizations = min_realization
         self._alpha = alpha
         self._std_cutoff = std_cutoff
-        self._analysis_iter_config = analysis_iter_config or AnalysisIterConfig()
         self._update_log_path = Path(update_log_path)
         self._min_realization = min_realization
+        self.num_iterations = num_iterations
```

Comment:
This is a bit annoying, as we dont know it up front, and it is only used in the export run_path workflow, and the state is kept in sync in `BaseRunModel`, not ideal. An option is to add a fixture which says the number of iterations, which will be more correct, but also add more over head. Not sure what the best action is.